code-studio/src/js/initApp/initApp.js

```diff
@@ -6,6 +6,11 @@ var chrome34Fix = require('./chrome34Fix');
 var loadApp = require('./loadApp');
 var project = require('./project');
 
+function isMobile() {
```

Comment:
It might be worth consolidating this with [apps/src/dom.js](https://github.com/code-dot-org/code-dot-org/blob/d8ae50cc91e1cbcb2e0a05879c7510661abcd176/apps/src/dom.js#L108-L108) eventually, or moving it to a common location.